src/SWP/Bundle/ContentBundle/Rule/Applicator/ArticleRuleApplicator.php

```diff
@@ -77,6 +88,10 @@ public function apply(RuleInterface $rule, RuleSubjectInterface $subject)
 
         $subject->setTemplateName($configuration[$this->supportedKeys[1]]);
 
+        if ((bool) $configuration[$this->supportedKeys[2]]) {
```

Comment:
there is option resolver which resolves the configuration array before it goes here and it makes sure that this key is always here.